Docs/Readme.md

```diff
@@ -0,0 +1,30 @@
+![W3C SVG Logo](https://www.w3.org/Icons/SVG/svg-logo-v.png)
```

Comment:
As in the above comment, the Readme stuff is a bit tricky
- Repository/Readme or README - Could provide a link to NuGet package and Documentation
- Documentation/Readme or Introduction - Should not provide a link to Documentation
- Nuget/Readme - Should not provide a link to NuGet package

I simply keep the three separate, as each content will evolve differently.

Reply:
Ah ok, so this one is used as the nuget readme?

Reply:
Yes, not yet but will be part of the release workflow update. 
The NuGet/Logo will also reference `Docs/images/svg.png` instead of the current one under `Source`.